bugsnag-android-core/src/test/java/com/bugsnag/android/EventDeliveryStrategyTest.kt

```diff
@@ -0,0 +1,116 @@
+package com.bugsnag.android
+
+import com.bugsnag.android.BugsnagTestUtils.generateImmutableConfig
+import com.bugsnag.android.internal.BackgroundTaskService
+import com.bugsnag.android.internal.StateObserver
+import com.bugsnag.android.internal.dag.ValueProvider
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.junit.MockitoJUnitRunner
+import java.util.Date
+
+@RunWith(MockitoJUnitRunner::class)
+internal class EventDeliveryStrategyTest {
+
+    @Mock
+    lateinit var eventStore: EventStore
+
+    private val apiKey = "BUGSNAG_API_KEY"
+    private val notifier = Notifier()
+    val config = generateImmutableConfig()
+    val callbackState = CallbackState()
+    private val logger = InterceptingLogger()
+    lateinit var deliveryDelegate: DeliveryDelegate
+    val handledState = SeverityReason.newInstance(
+        SeverityReason.REASON_UNHANDLED_EXCEPTION
+    )
+
+    private var testException: RuntimeException? = null
+    val event = Event(RuntimeException("Whoops!"), config, handledState, NoopLogger)
+
```

Comment:
these could all be `private`, the formatting should also be consistent (at minimum group `val`s together and `var`s together)